src/main/java/be/syntra/devshop/DevshopFront/services/utils/ProductMapper.java

```diff
@@ -32,18 +28,30 @@ public ProductDto convertToProductDto(Product product) {
                 .price(product.getPrice())
                 .description(product.getDescription())
                 .archived(product.isArchived())
-                .totalInCart(product.getTotalInCart())
                 .categoryNames(categoryMapper.mapToCategoryNames(product.getCategories()))
                 .averageRating(product.getAverageRating())
                 .ratings(product.getRatings())
                 .reviews(product.getReviews())
                 .build();
     }
 
+    public Product convertToProduct(ProductDto productDto) {
```

Comment:
this is the unused method I was talking about. Don't forget to remove the mapperfield + constructor

Reply:
removed